compiler/rustc_mir/src/transform/coverage/mod.rs

```diff
@@ -284,7 +284,7 @@ impl<'a, 'tcx> Instrumentor<'a, 'tcx> {
         let tcx = self.tcx;
         let source_map = tcx.sess.source_map();
         let body_span = self.body_span;
-        let file_name = Symbol::intern(&self.source_file.name.to_string());
+        let file_name = Symbol::intern(&self.source_file.name.prefer_remapped().to_string_lossy());
```

Comment:
Heads up, code coverage (@richkadel and @tmandry?), this PR will make source locations emitted here respect the [--remap-path-prefix](https://doc.rust-lang.org/rustc/command-line-arguments.html#--remap-path-prefix-remap-source-names-in-output) compiler option. Please let us know if there is a reason not to do this.

(Note: if the compiler is invoked without `--remap-path-prefix` then the behavior does not change)

Reply:
Thank you, @richkadel!